### DISCUSSION TITLE
Support http2 protocol

### DISCUSSION THREAD

**Discussion by aagumin** [mosecorg/mosec#567] [Q&A] [answered]
can i serve mosec with http2 support?

**Comment by kemingy** [comment]
This is an interesting quantion. I checked the `axum` and `hyper`. It should be easy to enable the HTTP/2 feature.

You can try the PR https://github.com/mosecorg/mosec/pull/568.

Hints: to build it locally, you need to install the rust-lang and run `make dev`.

**Reply by kemingy** [reply, marked as answer]
You can try the HTTP/2 with `pip install -U mosec==0.8.8`
